### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#21] [Show and tell]
I have regenerated the translation files to include the latest changes.  This has shown that the translations are not complete:
| Language | % completion |
|--------|--------|
| ar | 87.79% |
| bg | 95.05% |
| bn | 87.79% |
| bs | 90.76% |
| ca | 88.12% |
| cs | 99.01% |
| da | 92.74% |
| de | 94.06% |
| el | 95.05% |
| es | 88.12% |
| et | 88.12% |
| fa | 94.39% |
| fi | 98.68% |
| fr | 99.67% |
| gl | 95.05% |
| he | 95.38% |
| hu | 87.79% |
| id | 89.11% |
| it | 88.12% |
| ja | 91.09% |
| ka | 98.68% |
| ko | 92.08% |
| lt | 87.79% |
| lv | 98.68% |
| ms | 96.04% |
| nl | 88.12% |
| no | 92.74% |
| pl | 96.04% |
| pt | 86.47% |
| pt | 86.47% |
| ro | 88.12% |
| ru | 98.68% |
| sk | 88.12% |
| sl | 87.13% |
| sr | 90.10% |
| sv | 99.01% |
| ta | 87.79% |
| th | 88.12% |
| tr | 97.03% |
| uk | 98.35% |
| vi | 93.07% |
| zh | 99.01% |

Anyone able to translate to one of those languages is welcome to create a pull-request with translations (files are under adminer/lang).

Thanks in advance!

**Comment by hjsleijster** [comment]
I guess it's safe to say these translations are complete, because some words can't be or shouldn't be translated.
For example: database, mysql, data, 'HH:MM:SS'.

Is it possible to see the most recent added strings that should be translated?

**Comment by LionelLaffineur** [comment]
Dear @hjsleijster,
You can find the language files here : https://github.com/adminerevo/adminerevo/tree/main/adminer/lang.
If you are not too used to pull requests, you can post here the file after translations and I'll take care of adding it to the project repo.
Regards,
L.

**Comment by isaacpolaino** [comment]
[es.inc.php.txt](https://github.com/user-attachments/files/17790620/es.inc.php.txt)
Hi!
I'm glad Adminer is still alive. I thank you so much for keep it going, now as AdminerEvo.
I can't do anything with programming, but I can put my grain of sand with translation to Spanish.
I've never pulled on GitHub so I think it's better to place here the file than broke something :)
Regards.
Isaac

**Reply by LionelLaffineur** [reply]
Thanks for your contribution!

**Reply by mckaygerhard** [reply]
> I've never pulled on GitHub so I think it's better to place here the file than broke something

its fine people wants to contribute but we need also follow process that facilite and made easy the life of developers.. git and pull request is the facto and standard way of developers that uses linux and any project in git

this githhubcrap from M$ already has nice GUI (iu i guess is the best of all.. ) so usage is not so hard @isaacpolaino you must try it!

**Reply by isaacpolaino** [reply]
Sorry, @mckaygerhard. You are right. But I've really never used git. I only have time to try to do things with my phone while I go in the bus to my first job in the morning (I have another job later, none of them related with code) and it takes a long time for me to do a simple thing this way. I'm trying to learn but I think it will take me so much.  I accepted the offer from @LionelLaffineur to post here the file. I thought it was better this way than wait until 2025 Christmas... 😁😁😁

**Reply by mckaygerhard** [reply]
umm got it @isaacpolaino but a phone is not a developer tool unfortunatelly.. we must reconice that when we cannot do the things we cannot..  

phone helps so much people today.. but something it add a layer of complexity event help.. if you upload a file and nobody is in right moment to commited to git.. will not help so add another task to do.. 

immagine this same for more that one user.. also git is available for phone also.. learning is not so hard
